app/src/main/java/comp3350/mealbuddy/objects/LabelGoal.java

```diff
@@ -28,10 +28,14 @@ public String toString() {
 
     @Override
     public boolean equals(Object o) {
-        if (this == o) return true;
-        if (!(o instanceof LabelGoal)) return false;
-        LabelGoal labelGoal = (LabelGoal) o;
-        return id.equals(labelGoal.id);
```

Comment:
I changed it so we could search a Label list like get("<labelName>") instead of having to make a goal to search the list but it didn't end up working.  i will change it back. We'll have to see next iteration if we want to just keep going with what we have or refactor into something like making edible a composite object and then the day will just have 4 edible objects instead of lists of edibles.